rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java

```diff
@@ -491,7 +497,7 @@ public void onClick(ClickEvent event) {
     private ToolStripButton getCloudAutomationLinkButton() {
         ToolStripButton cloudAutomationButton = new ToolStripButton("Cloud Automation");
 
-	cloudAutomationButton.addClickHandler(new ClickHandler() {
+        cloudAutomationButton.addClickHandler(new ClickHandler() {
             public void onClick(ClickEvent event) {
                 Window.open("/cloud-automation", "", "");
```

Comment:
Ah yes, sorry, I was looking at the Javascript API :( 

http://www.w3schools.com/jsref/met_win_open.asp